packages/frontend/amp/components/Elements.tsx

```diff
@@ -25,11 +25,15 @@ import { MapEmbed } from '@frontend/amp/components/elements/MapEmbed';
 import { AudioAtom } from '@frontend/amp/components/elements/AudioAtom';
 
 export const Elements = (
-    elements: CAPIElement[],
+    elements: GenericElement[],
```

Comment:
Maybe some comments to explain what this 'GenericElement' is? With the removal of 'CAPIElement', the naming becomes too generic (notice my pun) to be self documenting.

Reply:
`UnknownElement` is an alternative perhaps? I agree the naming isn't great. Element itself is usually something I'd avoid too as too general, but it's used everywhere in the codebase here with a specific meaning so don't feel I can fight against that now.

Reply:
Or `CAPIElements` and `KnownCAPIElements` ?